1-exercises/E-for-loop/exercise1.js

```diff
@@ -7,8 +7,11 @@
 
 // Change the below code to use a for loop instead of a while loop.
 let i = 0;
-while(i < 26) {
+for (i < 26; i++;) {
     console.log(String.fromCharCode(97 + i));
     i++;
 }
+
```

Comment:
Hey! Thank you for showing another of doing this exercise. I actually used a for-loop.